store/localstore/txn.go

```diff
@@ -52,12 +52,12 @@ func newTxn(s *dbStore, ver kv.Version) *dbTxn {
 // Implement transaction interface
 
 func (txn *dbTxn) Get(k kv.Key) ([]byte, error) {
-	log.Debugf("[kv] get key:%q, txn:%d", k, txn.tid)
+	log.Debugf("[kv] get key:% x, txn:%d", k, txn.tid)
```

Comment:
It makes the hex number more readable, like `ff 00 ff 00` instead of `ff00ff00`.